code/modules/events/shuttle_loan.dm

```diff
@@ -7,14 +7,31 @@
 #define ITS_HIP_TO 7
 #define MY_GOD_JC 8
 
-
 /datum/round_event_control/shuttle_loan
 	name = "Shuttle Loan"
 	typepath = /datum/round_event/shuttle_loan
-	max_occurrences = 1
+	max_occurrences = 3
 	earliest_start = 7 MINUTES
 	category = EVENT_CATEGORY_BUREAUCRATIC
 	description = "If cargo accepts the offer, fills the shuttle with loot and/or enemies."
+	///The types of loan offers that the crew can recieve.
+	var/list/shuttle_loan_offers = list(HIJACK_SYNDIE, RUSKY_PARTY, SPIDER_GIFT, DEPARTMENT_RESUPPLY, ANTIDOTE_NEEDED, PIZZA_DELIVERY, ITS_HIP_TO, MY_GOD_JC)
```

Comment:
```suggestion
	var/list/shuttle_loan_offers = list(
	HIJACK_SYNDIE,
	RUSKY_PARTY,
	SPIDER_GIFT,
	DEPARTMENT_RESUPPLY,
	ANTIDOTE_NEEDED,
	PIZZA_DELIVERY,
	ITS_HIP_TO, 
	MY_GOD_JC,
	 )
```

May as well clean it up for diff-prettyness (and readability) in case someone wants to add a new shuttle loan offer event in the future. I'm just doing this as a suggestion edit, but would be epic if you could alphabetize this as well.